feat: expose more details in trivy_vulnerability_id metric

MiggL opened the pull request:
## Description
Extends the `trivy_vulnerability_id` metric to include `fixed_version`, `vuln_title`, and `vuln_score`

## Related issues
- Close #152 

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/trivy-operator/blob/main/CONTRIBUTING.md) to this repository.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've not updated the [documentation](https://github.com/aquasecurity/trivy-operator/tree/main/docs).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/aquasecurity/trivy-operator?pullRequest=962) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/aquasecurity/trivy-operator?pullRequest=962) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/aquasecurity/trivy-operator?pullRequest=962) it.</sub>

dkulchinsky commented:
@MiggL this is cool, I was going to suggest a similar one (was also thinking that `trivy_image_exposedsecrets` could be useful with a few more labels)

you'll probably want to adjust the tests in https://github.com/aquasecurity/trivy-operator/blob/main/pkg/metrics/collector_test.go accordingly, with the new labels.

chen-keinan commented:
@MiggL thank you for the contribution I have added two comments

chen-keinan commented:
@MiggL also need to update [metric collector test](https://github.com/aquasecurity/trivy-operator/blob/main/pkg/metrics/collector_test.go#L157) with new fields and [docs](https://github.com/aquasecurity/trivy-operator/blob/main/docs/tutorials/integrations/metrics.md)

MiggL commented:
@chen-keinan I've updated the metric collector test and docs now.

> @MiggL thank you for the contribution I have added two comments

What comments are you referring to?

chen-keinan commented:
@MiggL that's it nothing else, just please rebase your branch with upstream.

chen-keinan commented on pkg/metrics/collector.go:
```diff
@@ -430,11 +437,18 @@ func (c ResourcesMetricsCollector) collectVulnerabilityIdReports(ctx context.Con
+						vulnLabelValues[17] = strconv.FormatFloat(*vuln.Score, 'f', 1, 64)
+					} else {
+						vulnLabelValues[17] = ""
+					}
```
@MiggL just this nit: comment

MiggL commented:
All right. Fyi I also changed the third strconv.FormatFloat argument to -1 so that the string contains the least amount of decimals i.e. "6" instead of "6.0" but still "6.1"

MiggL commented:
@chen-keinan fixed

chen-keinan commented:
@MiggL LGTM 🚀
